kalsmic/iReporterApi
# E4218332 integrate edit comment specific

**This pull request implements edit comment of a red-flag or intervention record functionality:-**
Logic implemented
- Only user who created the record can edit it
- Admin cannot edit comment of an incident. The button is hidden automatically.
- To edit the record, click on view details, while on the details page, click on edit comment.

**User story** https://www.pivotaltracker.com/epic/show/4218332

**How to test**
- log in  as admin and view details of a red-flag or interventions, you will notice there is no update comment button
- log in as a non-admin user and view details of a red-flag or interventions. Try editing the comment of a red-flag or intervention record.
- log out and try accessing the profile page or any other protected page

**Links**
- Login page : https://kalsmic.github.io/iReporterApi/UI/index.html
- Sign up page:  https://kalsmic.github.io/iReporterApi/UI/user/signup.html
- Profile page : https://kalsmic.github.io/iReporterApi/UI/user/index.html

**Credentials**
- Username: `arthur2` Password: `Password123`
- Usernmae: `admin` Password: `iReporter123`

@NicholusMuwonge @Rhytah @JamesMudidi @sanya-kenneth @frankopkusianwar @AtamaZack @ManuelDominic @KabohaJeanMark @nadralia @reifred @llwasampijja @richien @Genza999 @3Nakajugo @e-ian @Manorlds-Eaglespark @PatrickMugayaJoel @bisonlou @v1b3m @mwinel @sebbss
